python/torch_mlir_e2e_test/test_suite/reduction.py

```diff
@@ -601,3 +601,61 @@ def forward(self, a):
 @register_test_case(module_factory=lambda: ReduceFrobeniusNormKeepDimModule())
 def ReduceFrobeniusNormKeepDimModule_basic(module, tu: TestUtils):
     module.forward(torch.rand(3, 4, 5))
+
+# ==============================================================================
+
+class MseLossNoReductionModule(torch.nn.Module):
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1 , -1], torch.float32, True),
+        ([-1 , -1], torch.float32, True),
+    ])
+
+    def forward(self, x, y):
+        return torch.ops.aten.mse_loss(x, y, reduction=0)
+
+@register_test_case(module_factory=lambda: MseLossNoReductionModule())
+def MseLossNoReductionModule_basic(module, tu: TestUtils):
+    module.forward(torch.randn(2, 4), torch.randn(2, 4))
```

Comment:
This applies to all three test cases: use `tu.rand` to generate tensors. See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#test-case-function

> all random inputs should be generated through the TestUtils object.